From edcc97e71bab4b8e5423059712091ce14c916086 Mon Sep 17 00:00:00 2001
From: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
Date: Mon, 16 Oct 2017 15:07:46 +0300
Subject: [PATCH 42/46] ARC: add asm/gpio.h to fix compilation error with
 CONFIG_CMD_GPIO

With CONFIG_CMD_GPIO compilation reports error.
common/cmd_gpio.c:13:22: fatal error: asm/gpio.h: No such file or directory
 #include <asm/gpio.h>
                      ^

Signed-off-by: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
---
 arch/arc/include/asm/gpio.h | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 arch/arc/include/asm/gpio.h

diff --git a/arch/arc/include/asm/gpio.h b/arch/arc/include/asm/gpio.h
new file mode 100644
index 0000000000..306ab4c9f2
--- /dev/null
+++ b/arch/arc/include/asm/gpio.h
@@ -0,0 +1 @@
+#include <asm-generic/gpio.h>
-- 
2.11.0

